core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -800,7 +800,7 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     awaitAssignment(consumer, partitions.toSet)
 
     val producer = createProducer()
-    val producerRecords = partitions.flatMap(sendRecords(producer, partitionCount, _))
+    val producerRecords = partitions.flatMap(sendRecords(producer, numRecords = 15, _))
```

Comment:
This test aims to send records to all the partitions (30). with this change, we only send to 15 partitions. maybe we need to enable debug logs to understand the root cause.

Reply:
hi @omkreddy , no, with this change, we won't **only sent to 15 partitions** . 

The variable `partitions` in this line, is the total partitions in all topics, that is, we have 3 topics, and 30 partitions for each in the test, and the variable `partitions` is with size of 90. And here, we send the records to all the 90 partitions with `numRecords` records for each.

So, in the test, we first collect all the partitions 
```
def testLowMaxFetchSizeForRequestAndPartition(): Unit = {
    val topic1 = "topic1"
    val topic2 = "topic2"
    val topic3 = "topic3"
    val partitionCount = 30
    val topics = Seq(topic1, topic2, topic3)
    ....
    // we collect all the TopicPartition info, that is, the partitions.size() will be 90
    val partitions = topics.flatMap { topic =>
      (0 until partitionCount).map(new TopicPartition(topic, _))
    }
    ....
    // so later, we send the records to all the 90 partitions with `numRecords` records for each. 
    // that is, the change to the number of the records sent won't affect the test itself
    val producerRecords = partitions.flatMap(sendRecords(producer, partitionCount, _))
   ...
}
```  
Thank you.

Reply:
Thanks for the explanation. I am still not sure, whey we are not able consume in 60seconds. Let us see, if can reproduce the issues with this PR.

Reply:
Sure. Thank you.